x-pack/libbeat/management/managerV2.go

```diff
@@ -618,6 +633,8 @@ func (cm *BeatV2Manager) reloadOutput(unit *client.Unit) error {
 }
 
 func (cm *BeatV2Manager) reloadInputs(inputUnits []*client.Unit) error {
+	cm.reloadLock.Lock()
+	defer cm.reloadLock.Unlock()
```

Comment:
Why do you need this if you are using an atomic? I also question if an atomic is even needed, isn't `reloadInputs` call from a single goroutine and it blocks until complete. So how can it be executed at the same time?

Reply:
You're right, I'll remove it.